### PR TITLE
Correct teleport start for db getting started

### PR DESCRIPTION
**Pull request by stevenGravy**
Corrected parameters to work with `teleport db start`

### CODE REVIEW

**Review by alexfornuto — APPROVED** [review]

**Comment by ptgott** [comment]
@stevenGravy I haven't tested this out, but looking at the source, it seems like the `db start` command has [all of these flags](https://github.com/gravitational/teleport/blob/166ef1adcce258abf348e9ce41e5dd5265487af8/tool/teleport/common/teleport.go#L203-L215) except for `db-name`—instead there's a `name` flag. Should we use that instead?

File: docs/pages/database-access/getting-started.mdx [hunk 1]
```diff
@@ -116,7 +116,8 @@ endpoint and region appropriately. The `--auth-server` flag must point to the
 address of your Teleport Proxy Service.
 
 ```code
-$ teleport db start \
+$ teleport start \
```
Comment (r0mant) [inline docs/pages/database-access/getting-started.mdx:119]: We do have a `teleport db start` command but its flags are `--name`, `--protocol`, `--uri` and `--aws-region` (i.e. without `--db` prefix.

**Comment by stevenGravy** [comment]
> @stevenGravy I haven't tested this out, but looking at the source, it seems like the `db start` command has [all of these flags](https://github.com/gravitational/teleport/blob/166ef1adcce258abf348e9ce41e5dd5265487af8/tool/teleport/common/teleport.go#L203-L215) except for `db-name`—instead there's a `name` flag. Should we use that instead?

I'd say just stay to this as it allows adding other roles like ssh.

**Comment by ptgott** [comment]
For me, if we want to show users the `teleport db start` command, the getting started guide for adding databases would be the place to do it. We do demonstrate this command in our Database Access guides, so it might also be a bit strange to see it there but not in the getting started guide.

Pages that mention `teleport db start`:

```
 $ grep -Rl "\$ teleport db start" docs/pages
docs/pages/database-access/getting-started.mdx
docs/pages/database-access/guides/mongodb-atlas.mdx
docs/pages/database-access/guides/cockroachdb-self-hosted.mdx
docs/pages/database-access/guides/elastic.mdx
docs/pages/database-access/guides/snowflake.mdx
docs/pages/database-access/guides/aws-cassandra-keyspaces.mdx
docs/pages/database-access/guides/mysql-self-hosted.mdx
docs/pages/database-access/guides/mongodb-self-hosted.mdx
docs/pages/database-access/guides/cassandra-self-hosted.mdx
docs/pages/database-access/guides/redis.mdx
docs/pages/database-access/guides/redis-cluster.mdx
docs/pages/database-access/guides/postgres-self-hosted.mdx
docs/pages/database-access/reference/cli.mdx
docs/pages/management/admin/self-signed-certs.mdx
```

File: docs/pages/database-access/getting-started.mdx (hunk 1 above)
Comment (stevenGravy) [reply docs/pages/database-access/getting-started.mdx]: @ptgott @r0mant changed to use the `teleport db start` parameters, please review and confirm

**Review by ptgott — APPROVED** [review]

**Pull request merged**